app/assets/javascripts/my_modules/protocols.js

```diff
@@ -81,7 +76,7 @@ function bindEditDueDateAjax() {
     })
     .on("ajax:error", function(ev2, data2, status2) {
       // Display errors if needed
-      $(this).renderFormErrors("my_module", data.responseJSON);
```

Comment:
It was like this before, and is needed because it is nested inside another AJAX listener which uses 'data', so it would get overriden.